src/test/java/com/artipie/YamlSettingsTest.java

```diff
@@ -46,12 +46,27 @@
  *
  * @since 0.1
  * @checkstyle MethodNameCheck (500 lines)
- * @todo #187:30min Add a test for layout structure, it can be either `flat` or `org`. If it's
- *  omitted, it should treated as a `flat` layout. See RepoLayout javadocs for more details.
  */
-@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.TooManyMethods"})
 class YamlSettingsTest {
 
+    @Test
+    public void shouldSetFlatAsDefaultLayout() throws Exception {
+        final YamlSettings settings = new YamlSettings(
+            String.join(
+                "",
+                "meta:\n",
+                "  storage:\n",
+                "    type: s3\n",
+                "    path: /var/artipie/configs\n"
+            )
+        );
+        MatcherAssert.assertThat(
+            settings.layout(),
+            Matchers.is("flat")
```

Comment:
@IlyaMoskva using `new IsEqual<>()` here will be more in line with code style of the project, see https://github.com/artipie/artipie/blob/master/CONTRIBUTING.md#code-style

Reply:
Changed.